docs/examples/databases/create-document.md

```
@@ -10,7 +10,13 @@ const result = await databases.createDocument({
     databaseId: '<DATABASE_ID>',
     collectionId: '<COLLECTION_ID>',
     documentId: '<DOCUMENT_ID>',
-    data: {},
+    data: {
+        "username": "walter.obrien",
+        "email": "walter.obrien@example.com",
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)